### ISSUE TITLE
Add autosave

### ISSUE THREAD

**Issue by richelbilderbeek** [richelbilderbeek/Brainweaver#11]
_From @richelbilderbeek on October 10, 2013 6:44_

Let all Set methods signal the AutoSave and let it create -say- 10 saving backups. 

_Copied from original issue: richelbilderbeek/ProjectRichelBilderbeek#112_

**Comment by richelbilderbeek** [comment]
https://github.com/richelbilderbeek/ProjectRichelBilderbeek/issues/133 gives an example how autosave fails at the current state

**Comment by richelbilderbeek** [comment]
When the code crashes, there should be a mechanism that has saved the work.

Hmmm, this should be simple: upon a normal close, the autosaved file is deleted. Upon a crash, that file is the one to be loaded.

Steps:
 * [ ] Automatically save after every command
 * [ ] Delete the autosaved file upon valid program completion
 * [ ] Recover an autosaved file if discover

These steps must be done before #79 , as this issue allows for unexpected crashes :-)

**Comment by richelbilderbeek** [comment]
Created `issue_11_1.sh` and `issue_11_2.sh`. The latter makes Brainweaver crash by making the QtConceptMap throw an exception by `F9`.

**Comment by richelbilderbeek** [comment]
This issue is dependent on #85 , as the QtHideAndShowDialog throws an exception in its destructor somehow.

**Comment by richelbilderbeek** [comment]
Found a simpler solution: just always save to `autosave1.cmp` and `autosave2.cmp`.

Works well, this is the recovered concept map:

![issue](https://cloud.githubusercontent.com/assets/2098230/17887278/03b9cca4-6926-11e6-9deb-e32f5f2f06cf.png)

Note that the arrow heads are not saved yet, but that is another issue.
